File: java/test/ExistsTest.java
package voxgig.solardemosdk.sdktest;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;

import voxgig.solardemosdk.core.SolardemoSDK;

public class ExistsTest {

  @Test
  public void testMode() {
    SolardemoSDK testsdk = SolardemoSDK.testSDK();
    assertNotNull(testsdk, "expected non-nil SDK");
  }
}

File: kotlin/test/ExistsTest.kt
package voxgig.solardemosdk.sdktest

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test

import voxgig.solardemosdk.core.SolardemoSDK

class ExistsTest {

  @Test
  fun testMode() {
    val testsdk = SolardemoSDK.testSDK()
    assertNotNull(testsdk, "expected non-nil SDK")
  }
}
